engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -1941,6 +1942,23 @@ static void LogFrameBufferError(GLenum status)
         context->m_ModificationVersion = dmMath::Max(0U, context->m_ModificationVersion);
     }
 
+    static HComputeProgram OpenGLNewComputeProgram(HContext _context, ShaderDesc::Shader* ddf)
+    {
+        assert(0 && "Not implemented!");
+        return 0;
+    }
+
+    static HProgram OpenGLNewProgramFromCompute(HContext context, HComputeProgram compute_program)
+    {
+        assert(0 && "Not implemented!");
+        return 0;
+    }
+
+    static void OpenGLDeleteComputeProgram(HComputeProgram prog)
+    {
+        assert(0 && "Not implemented!");
+    }
```

Comment:
None of these functions should be called from anywhere in the engine yet. The renderer checks for context support in the compute.cpp code, so for now this is fine.